smallcloudai/refact-chat-js
# Pin button

# 📍 messages



## Description

Adds support for opening / apply diffs from a 📍 message.
Also allow the ide to signal to chat that diffs have been applied
and lets the chat know about files open in the IDE.

## Type of change




## How to Test



1. Run a prompt that would cause a code change
2. next to the 📍 message there should be open + apply buttons
3. open, will open the file in an ide, apply will make the changes directly to the file system.


## Screenshots (if applicable)



![Screenshot description](url/to/screenshot.png)

## Checklist




## Linked Issues



- Fixes #123
- Closes #456

## Additional Notes
